packages/shared/client/ReplayClient.ts

```diff
@@ -228,6 +237,13 @@ export class ReplayClient implements ReplayClientInterface {
     return sessionId;
   }
 
+  async findAnnotations(listener: AnnotationListener, kind: string) {
```

Comment:
tiny nit: This feels like a backwards order of parameters for this sort of thing